R/shiny.R

```diff
@@ -0,0 +1,35 @@
+#' Open an interactive chat application
+#'
+#' Use `chatApp()` to launch an interactive shiny app for any chat instance.
+#'
+#' @param chat A chat object created by [new_chat_openai()] or friends.
+#' @export
+#' @returns A [shiny::shinyApp].
+chatApp <- function(chat) {
+  check_installed(c("bslib", "shiny", "shinychat"))
+
+  ui <- bslib::page_fluid(
+    shinychat::chat_ui("chat"),
+    shiny::actionButton(
```

Comment:
Not sure if this is the best way to add a close button but it's what claude suggested 😄